A small update to how we formulate our rules (#8)
mwelzl/draft-ccwg-ratelimited-increase: draft-welzl-ccwg-ratelimited-increase.md

Irrespective of the current state of a congestion control mechanism, senders of congestion controlled transport protocols:

> 1. MUST limit the growth of cwnd when FlightSize < cwnd in some way.

Author: ```suggestion
1. MUST include a way to limit the growth of cwnd when FlightSize < cwnd.
```

> 2. SHOULD limit the growth of cwnd when FlightSize < cwnd with inc(maxFS).

Reviewer: MUST with /some way/ sounds fuzzy.
I guess the nearest could be:
/MUST include a way to limit.../

Author: I like that!  I'll try this now and merge.  We should have a first version available soon, I think - we can always fine-tune the -01 draft much more  :-)
